test/markup/yaml/keys.txt

```diff
@@ -0,0 +1,22 @@
+someKey:
+  anotherKey: value
+
+some_key:
+  another_key: value
+
+some key:
+  another key: value
+
+"some key":
+  "another key": value
+
```

Comment:
Would be nice to see "some-key" and "some.key" since those were points of issue from before.

Reply:
👍 Added